pyproject.toml

```
@@ -92,11 +92,17 @@ setup-args = [
 
 [tool.ruff]
 exclude = [
```

```suggestion
extend-exclude = [
```
(to not overwrite default excluded paths)

---

I really don't think it's an issue for us, `ruff` respects (and prioritizes) `.gitignore` rules, so if there's something we're not committing due to those rules, it won't be touched by `ruff` either and if it is something we can commit, we can then decide if we want to make `ruff` ignore it or not.